bindata/network/multus/multus.yaml

```diff
@@ -84,6 +84,31 @@ data:
       echo "Failed to copy files in ${sourcedir} to $DESTINATION_DIRECTORY"
       exit 1
     fi
+  allowlistcopy.sh: |-
+    #!/bin/bash
+    set -e
+
+    ALLOWLIST_CONF=/entrypoint/allowlist.conf
+    DESTINATION_DIRECTORY=/host/etc/cni/tuning/
+
+    cp -f $ALLOWLIST_CONF $DESTINATION_DIRECTORY
+
+    if [ $? -eq 0 ]; then
+      echo "Successfully copied tuning-cni config $ALLOWLIST_CONF to $DESTINATION_DIRECTORY"
+    else
+      echo "Failed to copy tuning-cni config $ALLOWLIST_CONFto $DESTINATION_DIRECTORY"
+      exit 1
+    fi
+  allowlist.conf: |-
```

Comment:
should update the name of the configmap to not be `cni-binary-copy-script` since it's no longer just that

Reply:
Updated to: "cni-copy-resources"